static/js/load_report.js

```diff
@@ -77,6 +78,37 @@
 
 	}];
 
+	function changeDateAfter(d){
+		var y = parseInt(d.slice(0,4));
```

Comment:
Convert d to date and the get the year and month instead of using slice.

Reply:
ok, sir.